test/controllers/update/remove.confirm.statement.controller.spec.ts

```diff
@@ -0,0 +1,63 @@
+jest.mock("../../../src/utils/logger");
```

Comment:
can you add jest.mock("ioredis");
at the top?
not adding it causes the 'test was forced to quit after timeout' message when you run the tests